### DISCUSSION TITLE
Google Identity Services

### DISCUSSION THREAD

**Discussion by okimangelo** [NorthStarMarketing/sign-in-with-google#99] [Q&A]
Is this plugin affected by the new Google Identity Services?

https://developers.google.com/identity/gsi/web/guides/migration

**Comment by tarecord** [comment]
Thanks for bringing this up! It doesn't directly affect the plugin because it specifically relates to using the client library:
>If your client is using the Google API Client Library for JavaScript or other older libraries for authorization, see [Migrate to Google Identity Services](https://developers.google.com/identity/oauth2/web/guides/migration-to-gis) for more information.

Since the plugin isn't using the client library, the changes don't apply here. However, they do suggest updating to use a new sign in button which would be a good improvement to user experience. I'll look into that further for a future release.
